src/utils/tests/object-inspector.js

```diff
@@ -105,6 +105,25 @@ describe("object-inspector", () => {
       expect(paths).to.eql(["root/bar", "root/__proto__"]);
     });
 
+    it("window object", () => {
```

Comment:
Lets give this a clearer descriptions... something like `Sets default properties for the window object`?